src/core/gosling.schema.ts

```diff
@@ -728,71 +729,3 @@ export type TemplateTrackMappingDef = Omit<
 export type ChannelWithBase = Channel & {
     base?: string;
 };
-
-/* ----------------------- Below is deprecated ----------------------- */
```

Comment:
Not directly related to this PR, but removed this part since this is not used anymore.